metpy/calc/tools.py

```diff
@@ -269,7 +269,7 @@ def reduce_point_density(points, radius, priority=None):
     array([ True, False,  True])
     >>> metpy.calc.reduce_point_density(np.array([1, 2, 3]), 1.,
     ... priority=np.array([0.1, 0.9, 0.3]))
-    array([False,  True, False])
+    array([ False,  True, False])
```

Comment:
These are actually moving backwards. The failure was caused by the fact that the docs were building against NumPy 1.13.3, after which we had to change these examples to fix the failure. The real question is: why are we not getting the right version of numpy?